build.gradle

```diff
@@ -32,6 +35,24 @@ repositories {
     }
 }
 
+configurations.all {
```

Comment:
I'm not convinced this is required and the place you cribbed this from was using an older version of SoapUI, so this could cause issues.